bevy_rapier3d/examples/joints3.rs

```diff
@@ -40,11 +40,11 @@ fn enable_physics_profiling(mut pipeline: ResMut<PhysicsPipeline>) {
 fn setup_graphics(mut commands: Commands) {
     commands
         .spawn(LightComponents {
-            translation: Translation::new(1000.0, 100.0, 2000.0),
+            transform: Transform::from_translation(Vec3::new(1000.0, 100.0, 2000.0)),
             ..Default::default()
         })
         .spawn(Camera3dComponents {
-            transform: Transform::new_sync_disabled(Mat4::face_toward(
```

Comment:
This method was removed in Bevy 2.0.

Since the release blog post mentioned disabling sync was an unwanted workaround on the _old_ transform system, I guessed that it was no longer necessary and switched to `::new()`:

> Setting a Transform to a specific matrix value (ex: `Mat4::look_at()`) was extremely cumbersome, and the value would be immediately overwritten unless the user explicitly disabled component syncing.